_posts/2025-02-04-january-project-update.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "January 2025 Project Update"
+date: 2025-02-04
+categories: updates
+---
+
+We’re kicking off 2025 with exciting progress on Athena!
+This month, we launched the first public devnet, implemented the PROXY method, and introduced new token-related smart contracts.
+Let’s dive into the details.
+
+## First Public Devnet Released
+
+We’re thrilled to announce the release of Athena’s first public devnet!
```

Comment:
Link to configs? Or link how to join it?

Reply:
Could you pass a link to the docs explaining how to join the devnet?

Reply:
Added